setup.py

```diff
@@ -20,7 +20,6 @@
 
 class get_pybind_include(object):
     """Helper class to determine the pybind11 include path
-
```

Comment:
This blank line is supposed to be here (docstring = one-line summary + blank line + details).